src/modules/menu/components/search/SearchResultList.vue

```diff
@@ -54,18 +66,49 @@ export default {
 
 <style lang="scss" scoped>
 @import 'src/scss/media-query.mixin';
+@import 'src/scss/variables';
 
-#search-results {
-    position: absolute;
-    top: 100%;
-    left: 0%;
+#search-results-container {
+    position: fixed;
+    top: $header-height;
+    // 45vh (so that previews are visible), but on small screen min 20rem (ca. 3 lines per category)
+    height: min(calc(100vh - $header-height), max(20rem, 45vh));
+    left: 0;
     width: 100%;
-    max-height: calc(75vh - 3rem);
+    margin: 0; //overwrites a bootstrap property from parent module
+    pointer-events: none;
+}
+.search-results-dev-site-warning {
+    // Must be important as it needs to overrite the top defined in #search-results-container
+    top: $header-height + $dev-disclaimer-height !important;
+    height: min(calc(100vh - $header-height - $dev-disclaimer-height), max(20rem, 45vh)) !important;
```

Comment:
don't know if the `!important` is required, if so it's maybe because we used an ID to style the element. Maybe if you remove this ID definition and switch to a class based styling for both approach you might be able to get rid of those `!important` (which can be quite dangerous to use...)

Reply:
"SearchBar.vue" defines the bootstrap class input-group that defines some strange margines that I want to reset, thats why I used an ID (as an ID overwrites styles defined in a class). But this does not solve the problem as you mentioned, it only moves it.
So now I changed it into a class, removed the two important rules that you mentioned, but added one important rule for the margin instead. This is maybe a slight improvement.